logging-cfg-local.yml

```diff
@@ -1,13 +1,13 @@
 version: 1
-disable_existing_loggers: False # this allow to get logger at module level
+disable_existing_loggers: false # this allow to get logger at module level
```

Comment:
Interesting. I think this was upper case because this was read by python? You shouldn't be needing ansible-lint here?

Reply:
I was checking that : the default for yaml boolean capitalization for yaml linter is all lowercase. I would be in favor of keeping it as Sami did, since it would bring the config file to a standard value.

With that being said, it would be best to also do so in the base template and be consistent across all repos, but those are small changes.

Reply:
Oh, that makes sense. I just ran lint on the whole project, I'm assuming then that you only run it on code files?